src/hooks/useTaskManager.ts

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from 'react';
+import { Task } from '../utils/types';
+import {
+  addTask,
+  deleteAllTasks,
+  deleteSingleTask,
+  getTasks,
+  updateTask,
+} from '../utils/api';
+
+function useTaskManager() {
+  const [tasks, setTasks] = useState<Task[]>([]);
+
+  useEffect(() => {
+    fetchTasks()
+      .then(() => {})
```

Comment:
Question 🤔: you're not doing anything after you fetch the data?

Reply:
Originally had it to wait for fetchTasks to run on mount and thought I needed .then() block. Removed now as a part of following comment response: [Review Comment for fetchTasks()](https://github.com/mannyv123/todo-list/pull/17#discussion_r1310951284)